docarray/typing/tensor/audio/abstract_audio_tensor.py

```diff
@@ -20,28 +19,36 @@ def to_bytes(self):
         tensor = (tensor * MAX_INT_16).astype('<h')
         return tensor.tobytes()
 
-    def save_to_wav_file(
+    def save(
         self: 'T',
         file_path: Union[str, BinaryIO],
-        sample_rate: int = 44100,
+        format: str = 'wav',
+        frame_rate: int = 44100,
         sample_width: int = 2,
+        *args,
```

Comment:
I know we had this pattern a lot in v1, but I am not sure i lile it tbh.

What about having a `pydub_args: Dict[str, Any]` parameters

and do segment.export(..., **pydub_arg)

Reply:
@JohannesMessner what do u think ?